MC/run/PWGHF/run_anchor_ini_from_cvmfs_example.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+export ALIEN_JDL_LPMANCHORPASSNAME="apass4"
+export ALIEN_JDL_MCANCHOR="apass4"
+export ALIEN_JDL_COLLISIONSYSTEM="pp"
+export ALIEN_JDL_LPMPASSNAME="apass4"
+export ALIEN_JDL_LPMRUNNUMBER="539071"
+export ALIEN_JDL_LPMPRODUCTIONTYPE="MC"
+export ALIEN_JDL_LPMINTERACTIONTYPE="pp"
+export ALIEN_JDL_LPMPRODUCTIONTAG="LHC24_2023zg_apass4_MC_test"
+export ALIEN_JDL_LPMANCHORRUN="539071"
+export ALIEN_JDL_LPMANCHORPRODUCTION="LHC23zg"
+export ALIEN_JDL_LPMANCHORYEAR="2023"
+
+# added export
+export NTIMEFRAMES=8
+export NSIGEVENTS=100
+export SPLITID=100
+export PRODSPLIT=153
+export CYCLE=0
+export ALIEN_PROC_ID=2963436952
+
+# disable the QC
+export DISABLE_QC=1
+
+# modify ini file, to have external generator and/or config from a specific tag different from the one used for anchoring
+ORIGINALINI=${O2DPG_ROOT}/MC/config/PWGHF/ini/GeneratorHF_D2H_bbbar_Bforced_gap5_Mode2.ini # original .ini file to be modified
```

Comment:
Additionally, some of the older tags don’t exist in this configuration. Perhaps we're using an old template?
```suggestion
ORIGINALINI=${O2DPG_ROOT}/MC/config/PWGHF/ini/GeneratorHF_D2H_bbbar_gap5.ini# original .ini file to be modified
```

Reply:
I would prefer to keep it like this (it is only an example). But of course it can be applied to any `.ini` file, in case the one we want to modify is not included (one just have to be careful that the other `ini` is as desired, except for the parts that are modified).